CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### 1.4.0 (Unreleased)
+
+[#270](https://github.com/marmelab/EventDrops/pull/270): Added `onMouseOver`, `onMouseOut` and `onClick` to label text.
```

Comment:
we'll update the changelog when we package the release. Please revert the change in that file.

Reply:
This is for the labels, not the drops. So for example, if one has a very long label name, one can add a tooltip to the labels to have a better description of it.